### DISCUSSION TITLE
Please be patient.

### DISCUSSION THREAD

**Discussion by therealjohndion** [evennia/evennia#3321] [Evennia Questions] [answered]
I am very new to this. I installed evennia on linux and I would like the  web interface to be accesible to my domain. Its only listening on localhost and not mydomain.com:4005 how do I make it listen on all interfaces? I want my friends to access the panel and be able to play in the browser! Thank you so much! This is great i am going to learn python now.

**Comment by therealjohndion** [comment]
this is what i have added in settings.py

https://termbin.com/trt4o

**Comment by volundmush** [comment]
This depends a lot on where and how you run it, but...

By default, Evennia's external-facing ports (4000, 4001, 4002, 4003, and 4004) bind to interface "0.0.0.0", which the system interprets to mean "all interfaces." So, it SHOULD already be accessible. You can check how this is done by looking at https://github.com/evennia/evennia/blob/main/evennia/settings_default.py

Depending on the host though, you may need to enable port forwarding or configure your firewall to allow it through. Where are you running Evennia? What does your domain point at? If you can provide some more details we'll be able to provide even better guidance.

**Comment by therealjohndion** [comment]
Hi! I already forwarded the port 4005 and  4000 I can hit 4000 but not 4005 http://unixhideout.solutions:4005 is what i am trying to achieve. You can telnet to 4000 but 4005 does not work.

**Reply by volundmush** [reply, marked as answer]
As a note, the webserver works a little differently.

```python
# The webserver sits behind a Portal proxy. This is a list
# of tuples (proxyport,serverport) used. The proxyports are what
# the Portal proxy presents to the world. The serverports are
# the internal ports the proxy uses to forward data to the Server-side
# webserver (these should not be publicly open)
WEBSERVER_PORTS = [(4001, 4005)]
```

Port 4001 is the external-facing port. 4005 should NOT be accessible to the outside world. So it's good that 4005 isn't responding, but why isn't 4001? Hmmm.

**Reply by therealjohndion** [reply]
OH! I didnt know it was 4001 let me try something. Thanks for your help ill let you know in a minute.

**Comment by therealjohndion** [comment]
https://canyouseeme.org/  port 4000 hits but 4005 gets connection refused which means my machine is not listening the port is closed to wan.

**Comment by therealjohndion** [comment]
It works! Thank you so much! Such quick help this is going to be great thank you for this soft ware!

**Comment by volundmush** [comment]
Glad that got you up and running! Note that we have a Discord at https://discord.gg/AJJpcRUhtF

A lot of great discussions happen here in real-time with developers working on their projects. You'll probably benefit greatly if you plan on using Evennia to create a game.

**Reply by therealjohndion** [reply]
excellent im on my way!

**Comment by therealjohndion** [comment]
Sorry it says invite invalid.

**Reply by volundmush** [reply]
Odd. Try this one... https://discord.gg/nfVg2J8vWf

**Reply by therealjohndion** [reply]
thanks

**Comment by therealjohndion** [comment]
![Pasted image](https://github.com/evennia/evennia/assets/83326035/1796c53a-d82f-4d12-8329-40d8cc1636e8)

**Reply by therealjohndion** [reply]
I figured it out im in thanks.

**Reply by KevesDev** [reply]
How'd you figure it out? I'm having the same issue.

Edit: Ah, I got in using my phone.
